### ISSUE TITLE
Inconsistent handling of search queries by find_census_vectors

### ISSUE THREAD

**Issue by baranovskypd** [mountainMath/cancensus#150]
(1) In find_census_vectors(), `type = c("male", "female")` **sometimes** works, but throws a warning. Try: 

```
find_census_vectors("full year, full time", "CA16",
                    type = c("male", "female"),
                    query_type = "exact")
```

Could you maybe remove the warning? Just please don't remove the ability to return data for both genders with one command, it is very handy.

(2) And sometimes `type = c("male", "female")` doesn't work: 

```
find_census_vectors("part year and/or part time", 
                    "CA16",
                    type = c("male", "female"),
                    query_type = "exact")
```
Says '_No exact matches found. Please check spelling and try again or consider using semantic or keyword search_.' However, this works (note exact same query term): 

```
find_census_vectors("part year and/or part time", 
                    "CA16",
                    type = "male",
                    query_type = "exact")
```
Could you please make this consistent _and_ allow to set `type = c("male", "female")`?

**Comment by dshkol** [comment]
Thanks for flagging this. Apologies, thought I had written a response earlier. Will look into this asap - agree should be addressed.

**Comment by dshkol** [comment]
Fixed in #152 - thanks again for the heads up @baranovskypd.

**Comment by baranovskypd** [comment]
You are welcome. Thanks for the quick update, and keep up the great work!
